src/index.d.ts

```diff
@@ -12,7 +12,7 @@ declare namespace preact {
 
 	interface VNode<P = {}> {
 		type: ComponentType<P> | string | null;
-		props: P & { children: ComponentChildren } | string | number | null;
```

Comment:
I don't think we expose Text VNodes (the only VNodes where props isn't an object) to the user, so I removed them.

Reply:
Good point :+1: